README.md

```diff
@@ -8,8 +8,8 @@ Odoo Connector Modules
 
 This OpenERP add-on has a modular and generic core, with the ability to be extended with additional modules for new features or customizations.
 
-The team for the community is here: https://launchpad.net/~openerp-connector-community
-Please join and subscribe to the mailing list.
+Feel free to subscribe on the mailing list:
```

Comment:
You should specify in which list inside the page (explicit better than implicit).